### ISSUE TITLE
Add accessible labels to buttons on home screen

### ISSUE THREAD

**Issue by CR96** [WildAid/o-fish-android#411]
**Is your feature request related to a problem? Please describe.**
Buttons should have descriptive labels for Android TalkBack users.

**Describe the solution you'd like**
Add labels to Account and Board Vessel buttons.
